setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup
+setup(
+    name='armus',
+    packages=['armus'],
```

Comment:
I think you need to do something to tell setup.py about the `create_migration` script, if you want pip to install it correctly